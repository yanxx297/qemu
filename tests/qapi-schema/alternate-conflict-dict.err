tests/qapi-schema/alternate-conflict-dict.json:6: Anonymous union 'MyUnion' member 'two' can't be distinguished from member 'one'
