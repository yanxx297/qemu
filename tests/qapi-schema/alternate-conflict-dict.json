# we reject alternates with multiple object branches
{ 'type': 'One',
  'data': { 'name': 'str' } }
{ 'type': 'Two',
  'data': { 'value': 'int' } }
{ 'alternate': 'Alt',
  'data': { 'one': 'One',
            'two': 'Two' } }
