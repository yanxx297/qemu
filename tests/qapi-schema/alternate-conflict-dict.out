[OrderedDict([('type', 'One'), ('data', OrderedDict([('name', 'str')]))]),
 OrderedDict([('type', 'Two'), ('data', OrderedDict([('value', 'int')]))]),
 OrderedDict([('union', 'MyUnion'), ('discriminator', OrderedDict()), ('data', OrderedDict([('one', 'One'), ('two', 'Two')]))])]
[{'enum_name': 'MyUnionKind', 'enum_values': None}]
[OrderedDict([('type', 'One'), ('data', OrderedDict([('name', 'str')]))]),
 OrderedDict([('type', 'Two'), ('data', OrderedDict([('value', 'int')]))])]
